addons/cargo/XEH_postInit.sqf

```diff
@@ -36,6 +36,15 @@
     };
 }] call CBA_fnc_addEventHandler;
 
+[QGVAR(unloadedCargoOnKilled), {
+    params ["_item", "_vehicle"];
+    private _velocity = velocity _vehicle apply {_x / CARGO_TUMBLE_VELOCITY_DIVISOR};
+    private _torque = _vehicle vectorModelToWorld [random CARGO_TUMBLE_TORQUE, 0, 0];
+
+    _item setVelocity _velocity;
+    _item addTorque _torque;
```

Comment:
```suggestion
    params ["_item", "_vehicle"];

    // Get direction from vehicle to item
    private _vectorDir = (getPosWorld _vehicle) vectorFromTo (getPosWorld _item);

    // Give some z
    _vectorDir set [2, random [0.5, 0.75, 1]];

    // Add some randomness to resulting velocity
    _vectorDir = (_vectorDir vectorMultiply random 8) vectorAdd (velocity _vehicle);
        
    _item setVelocity _vectorDir;
    _item addTorque [random CARGO_TUMBLE_TORQUE, random CARGO_TUMBLE_TORQUE, random CARGO_TUMBLE_TORQUE];
```

8 is a magical number that needs to be put into `script_component.hpp`. It comes from testing - imo 10 is slightly too much.

@Drofseh Please try code out before committing, I'm looking for feedback.